File: project/DAL/Book.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL
{
    public class Book
    {

        public int Id { get; set; }
        public string Title { get; set; }
        public string Summary { get; set; }
        public string Author { get; set; }//לא חייב לשים
        public int PageCount { get; set; }//חייב לשים
        public int? CategoryID { get; set; }//חייב לשים
        public virtual Category Category { get; set; }
        public virtual List<Lendings> Lendings { get; set; }

    }
}

File: client/src/app/classes/book.ts
// import { AotCompiler } from '@angular/compiler';

export class Book {
    // constructor(id?: number, title?: string,
    //      author?: string, summary?: string, ageCategory?: number, pageCount?: number) {
    constructor(id?: number, title?: string,
        author?: string, summary?: string, ageCategory?: number, pageCount?: number) {
        this.ageCategory = ageCategory;
        this.author = author;
        this.id = id;
        this.pageCount = pageCount;
        this.summary = summary;
        this.title = title;
    }
    id: number;
    title: string;
    author: string;
    summary: string;
    ageCategory: number;
    pageCount: number;
}
